veloroutes_voies_vertes/actions.py

```diff
@@ -40,3 +41,63 @@ def split_segment(*args):
         iface.messageBar().pushInfo('Véloroutes', msg)
     except QgsProviderConnectionException as e:
         iface.messageBar().pushCritical('Véloroutes', e.args[0])
+
+
+def remove_selected_portion(*args):
+    # get arguments
+    id_iti = int(args[0])
+
+    # get project instance
+    project = QgsProject.instance()
+
+    # get layers needed
+    layer_portions = project.mapLayersByName('v_portion')
+
+    # Control if layers exists
+    if not len(layer_portions):
+        iface.messageBar().pushCritical('Véloroutes', 'La couche v_portion n\'a pas été trouvée')
+        return
+    pLayer = layer_portions[0]
```

Comment:
```suggestion
    layer_portions = layer_portions[0]
```
Nommage des variables